ryhope/src/storage/pgsql/mod.rs

```diff
@@ -24,6 +24,8 @@ use tracing::*;
 
 mod storages;
 
+const MAX_PGSQL_BIGINT: i64 = 9223372036854775807;
```

Comment:
FYI: this is also a constant in [the stdlib](https://doc.rust-lang.org/std/primitive.i64.html#associatedconstant.MAX)